src/components/TimeDisplay.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components'
+
+const TimeDiv = styled.div`
+  padding: 2em 0;
+  text-align: center;
+  margin: 1em;
+  border-radius: 3px;
+  font-weight: 600;
+  background-color: ${p => p.theme.colors.primary};
+`
+
+const TimeText = styled.span`
+  color: ${p => p.theme.colors.text};
+  width: 6vw;
+  font-size: 4em;
+`
+
+const TimeUnitText = styled(TimeText)`
+  font-size: 1em;
+  margin: 0 10px;
+`
+
+const Separator = styled.span`
+  width: 20px;
+  font-size: 4em;
+`
+
+const TimeFlex = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const TimeDisplay = ({ days, hours, minutes, seconds }) => {
```

Comment:
thoughts on passing a timestamp to count down to here? Or do you want to do that in a container instead?

Reply:
Was thinking of doing that in the container